packages/lib/internal/fetcher.ts

```diff
@@ -1,3 +1,6 @@
+import http from "http";
+import https from "https";
```

Comment:
Import just the `Agent` from both packages. That should reduce bundle size by not importing the entire `http/https` package.

```suggestion
import { Agent as httpsAgent } from "https";
```